Hey I added some handle gizmos to the mod, and also numeric input support

CarnationRED opened the pull request:
Both numeric input and handles can be toggled on or off. By the way I added a tip to the edit panel, it appears when you drag handles.
**Mod is tested and works good  on Windows, I made assetbundles for Linux and OSX but I don't have those devices, so some testings are appreciated.**
Some wip demos here:
![test3](https://user-images.githubusercontent.com/46291383/85850618-bb870700-b7df-11ea-9a1b-1c5e3d6a41f2.gif)
![test2](https://user-images.githubusercontent.com/46291383/85850661-d0639a80-b7df-11ea-86eb-c37a2aa72fe3.gif)

Rafterman82 committed:
Merge pull request #3 from jrodrigv/master

For KSP 1.8.1

Rafterman82 committed:
Update 1.9.1

CarnationRED committed:
Add shape adjustment handles, add numeric input

Added 9 handles for  adjusting length, width and leading edges
Added numeric input support

CarnationRED committed:
Add shape adjustment handles, add numeric input

Added 9 handles for  adjusting length, width and leading edges
Added numeric input support

CarnationRED commented:
I found an issue, this does not work well on control surfaces.

jrodrigv commented:
This is awesome!

I will integrate the changes and do some testing!

El vie., 26 jun. 2020 a las 17:02, CarnationRED (<notifications@github.com>)
escribió:

> I found an issue, this does not work well on control surfaces.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jrodrigv/B9-PWings-Fork/pull/2#issuecomment-650227629>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB7IR64EMVJY3JEAKQ5C7JDRYS2CBANCNFSM4OJHKLNQ>
> .
>

CarnationRED committed:
Fix multi-part editing, ctrlsurf editing, add offset editing

Bug fix:
1. all wing parts are modified using handles
2. buggy support on control surfaces
3. dragging is not comfy
4. root/tip offsets are not modified when dragging width handles

Added capability to adjust the all-moving wing's rotating hinge position

CarnationRED committed:
Small tweaks on UI/control/handle model
